docs/installation/openshift/Deploying_openshift-acme_on_OpenShift.md

```diff
@@ -0,0 +1,108 @@
+Deploying openshift-acme on OpenShift
+=====================================
+
+## Overview
+
+This document describes the process to deploy [openshift-acme](https://github.com/tnozicka/openshift-acme)
+to issue certificates for applications on OpenShift automatically
+using [PKI ACME Responder](Deploying_PKI_ACME_Responder_on_OpenShift.md).
+
+## Deploying openshift-acme with PKI ACME Responder
+
+To deploy openshift-acme, get the source with the following commands:
+
+```
+$ git clone https://github.com/tnozicka/openshift-acme.git
+$ cd openshift-acme
+```
+
+A sample openshift-acme configuration for a single namespace is available at:
+
+- [deploy/single-namespace/role.yaml](https://github.com/tnozicka/openshift-acme/blob/master/deploy/single-namespace/role.yaml)
+- [deploy/single-namespace/serviceaccount.yaml](https://github.com/tnozicka/openshift-acme/blob/master/deploy/single-namespace/serviceaccount.yaml)
+- [deploy/single-namespace/issuer-letsencrypt-staging.yaml](https://github.com/tnozicka/openshift-acme/blob/master/deploy/single-namespace/issuer-letsencrypt-staging.yaml)
+- [deploy/single-namespace/deployment.yaml](https://github.com/tnozicka/openshift-acme/blob/master/deploy/single-namespace/deployment.yaml)
+
+Create a new issuer configuration (e.g. deploy/single-namespace/issuer-pki-demo.yaml)
+based on an existing one (e.g. deploy/single-namespace/issuer-letsencrypt-staging.yaml),
+and configure the **directoryUrl** to point to PKI ACME Responder (e.g. https://pki.demo.dogtagpki.org/acme/directory):
```

Comment:
to _your deployed_ PKI ACME Responder. 

We don't want people using our demo in production. :)

Reply:
OK, let me revise this to clarify that the URL points to a demo instance. I think we want to promote the demo so we can get some feedback from other users.